src/NetworkTest/testQuality/helpers/subscriberMOS.ts

```diff
@@ -119,7 +119,7 @@ export default function subscriberMOS(
          * We know that we're receiving "faulty" stats when we see a negative
          * value for bytesReceived.
          */
-        if (stats.audio.bytesReceived < 0 || stats.video.bytesReceived < 0) {
+        if (stats.audio.bytesReceived < 0 || stats.video && stats.video.bytesReceived < 0) {
```

Comment:
Can use `getOr`from `util` here instead of checking for the `video` property first.